stdlib/2and3/distutils/cmd.pyi

```diff
@@ -13,3 +13,28 @@ class Command:
     def finalize_options(self) -> None: ...
     @abstractmethod
     def run(self) -> None: ...
+
+    def announce(self, msg: str, level: int = ...) -> None: ...
+    def debug_print(self, msg: str) -> None: ...
+
+    def ensure_string(self, option: str, default: Optional[str]) -> None: ...
+    def ensure_string_list(self, option: str) -> None: ...
+    def ensure_filename(self, option: str) -> None: ...
+    def ensure_dirname(self, option: str) -> None: ...
+
+    def get_command_name(self) -> str: ...
+    def set_undefined_options(self, src_cmd: str, *option_pairs: Tuple[str, str]) -> None: ...
+    def get_finalized_command(self, command: str, create: int = ...) -> Command: ...
+    def reinitialize_command(self, command: Union[Command, str], reinit_subcommands: int = ...) -> Command: ...
+    def run_command(self, command: str) -> None: ...
+    def get_sub_commands(self) -> List[str]: ...
+
+    def warn(self, msg: str) -> None: ...
+    def execute(self, func: Callable[..., Any], args: Iterable[Any], msg: Optional[str] = ..., level: int = ...) -> None: ...
+    def mkpath(self, name: str, mode: int = ...) -> None: ...
+    def copy_file(self, infile: str, outfile: str, preserve_mode: int = ..., preserve_times: int = ..., link: Optional[str]= ..., level: Any = ...) -> Tuple[str, bool]: ...  # level is not used
+    def copy_tree(self, infile: str, outfile: str, preserve_mode: int = ..., preserve_times: int = ..., preserve_symlinks: int = ..., level: Any = ...) -> List[str]: ...  # level is not used
+    def move_file(self, src: str, dest: str, level: Any = ...) -> str: ...  # level is not used
+    def spawn(self, cmd: List[str], search_path: int = ..., level: Any = ...) -> None: ...  # level is not used
+    def make_archive(self, base_name: str, format: str, root_dir: Optional[str] = ..., base_dir: Optional[str] = ..., owner: Optional[str] = ..., group: Optional[str] = ...) -> str: ...
+    def make_file(self, infiles: Union[str, List[str], Tuple[str]], outfile: str, func: Callable[..., Any], args: List[Any], exec_msg: Optional[str] = ..., skip_msg: Optional[str] = ..., level: Any = ...) -> None: ...  # level is not used
```

Comment:
Can infiles be a more general type like `Iterable[str]`? Haven't traced the code to check.

Reply:
The function raises a `TypeError` if not those types:
https://github.com/python/cpython/blob/ede2ac913eba47131ee1bbc37a9aea344d678576/Lib/distutils/cmd.py#L386